index.bs

```diff
@@ -1355,8 +1355,8 @@ The formats of these signatures, as well as the procedures for generating them,
 
 The <dfn>authenticator data</dfn> structure encodes contextual bindings made by the [=authenticator=]. These bindings are
 controlled by the authenticator itself, and derive their trust from the [=[RP]=]'s assessment of the security properties of the
-authenticator. In one extreme case, the authenticator may be embedded in the client, and its bindings may be no more trustworthy
-than the [=client data=]. At the other extreme, the authenticator may be a discrete entity with high-security hardware and
+authenticator. In one extreme case, the authenticator MAY be embedded in the client, and its bindings MAY be no more trustworthy
+than the [=client data=]. At the other extreme, the authenticator MAY be a discrete entity with high-security hardware and
```

Comment:
Back out all the changes in this sentence.

Reply:
done.